kong/cmd/config.lua

```diff
@@ -49,19 +49,17 @@ local function db_export(filename, conf)
 end
 
 
-local function generate_init()
-  if pl_file.access_time(INIT_FILE) then
-    error(INIT_FILE .. " already exists in the current directory.\n" ..
-          "Will not overwrite it.")
+local function generate_init(filename)
+  if pl_file.access_time(filename) then
+    error(filename .. " already exists.\nWill not overwrite it.")
```

Comment:
Now this command accepts arguments such as: `$ kong config init ../kong.yml`, which would result in errors such as:
```
$ kong config init ../../../kong2.yml
Error: ../../../kong.yml already exists.
Will not overwrite it.

  Run with --v (verbose) or --vv (debug) for more details
```

Or even worse, in this case, `..` is my local `kong` repository:
```
$ kong config init ..          
Error: .. already exists.
Will not overwrite it.

  Run with --v (verbose) or --vv (debug) for more details
```

I'd suggest resolving the absolute path to the target config file before calling `generate_init()`, with, e.g., [pl_path.abspath()](http://stevedonovan.github.io/Penlight/api/libraries/pl.path.html#abspath).

Reply:
I can do that if it's a blocker, but I just see absolutely no problems with the error messages above, since they refer to paths as I gave as input. I think it's in fact very uncommon for CLI tools to resolve absolute paths in error messages:

```
$ gcc ../../foo.c
gcc: error: ../../foo.c: No such file or directory
gcc: fatal error: no input files
compilation terminated.

$ hexdump ../../bla
hexdump: ../../bla: No such file or directory
hexdump: all input file arguments failed
```

(Also, the other subcommands in `kong config` don't resolve absolute paths either.)